src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+body {
+  margin: 0%;
+  line-height: 60px;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
```

Comment:
we don't really neet to set `flex-direction` to `row`, because it's like that initially 😄